bin/narn.js

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env node
 const { spawn } = require("child_process");
-const { detectYarn, getYarnArgs, getNpmArgs } = require("../lib/narn-lib.js");
+const {
+  detectNpm,
+  getYarnArgs,
+  getNpmArgs,
+  detectPnpm
+} = require("../lib/narn-lib.js");
```

Comment:
I changed the detectYarn function to detectNpm / detect Pnpm since that was what they were actually doing